stock_pull_list/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+The pull list checks the stock situation at the given location and calculates
```

Comment:
My question here: in your flows, do you generate procurements that may generate purchases or just moves from other locations ? @EvaSForgeFlow @LoisRForgeFlow 

Because it makes me think about this : https://github.com/OCA/stock-logistics-orderpoint/pull/3

Reply:
It looks like it's serving the same purpose but with a selective approach. https://github.com/OCA/stock-logistics-orderpoint/pull/3 replenish all products with a need on the monitored location either automatically as soon as there is a new need, or manually by launching the replenishment action. From what I see, this one allows to choose which product to replenish in a wizard. Moreover, you can set a date boundary. It's the "select" trigger mode you don't have in https://github.com/OCA/stock-logistics-orderpoint/pull/3
Would be interesting to understand in which context this module is used.